addons/test_mail_full/tests/test_portal.py

```diff
@@ -162,3 +163,111 @@ def test_portal_mixin(self):
 
         record_portal._portal_ensure_token()
         self.assertTrue(record_portal.access_token)
+
+
+@tagged('portal')
+class TestPortalFlow(MailCommon, HttpCase):
+    """Portal flow test: share a link by email to a customer without an account for viewing a record through the portal.
```

Comment:
This is already in the class name
```suggestion
    """Share a link by email to a customer without an account for viewing a record through the portal.
```

Reply:
Yes indeed, thanks.